<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TNLocaleStorageSettingWidget</class>
 <widget class="QWidget" name="TNLocaleStorageSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TNLocaleStorageSettingWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_13">
   <property name="topMargin">
    <number>38</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,0,0,0,0,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="localStorageSystemTitle">
             <property name="minimumSize">
              <size>
               <width>78</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>系统设置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="bootUpCheckBox">
             <property name="text">
              <string>开机启动</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="noDisturbCheckBox">
             <property name="text">
              <string>免打扰</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_13">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="autoLoginCheckBox">
             <property name="text">
              <string>自动登录</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="safeLoginWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="bottomMargin">
               <number>28</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QLabel" name="localStorageSystemTitle_2">
                  <property name="minimumSize">
                   <size>
                    <width>112</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>安全登录</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="faceIdButton">
                    <property name="text">
                     <string>启用人脸识别</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="facelabel">
                <property name="text">
                 <string>密码登录后可使用人脸识别</string>
                </property>
                <property name="indent">
                 <number>118</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="localStorageSystemTitle_3">
             <property name="text">
              <string>语言设置</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="bottomMargin">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="localStorageSystemTitle_4">
               <property name="minimumSize">
                <size>
                 <width>112</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>语言</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="languageComboBox"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="localStorageLabelTitle">
             <property name="text">
              <string>本地存储设置</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,1">
             <property name="topMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="labelFileMan">
               <property name="minimumSize">
                <size>
                 <width>112</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>文件管理</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="localStorageFileLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>120</number>
             </property>
             <item>
              <widget class="QLabel" name="localStorageLabel">
               <property name="text">
                <string>默认把接收到的文件保存到此文件夹中</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>120</number>
             </property>
             <property name="bottomMargin">
              <number>30</number>
             </property>
             <item>
              <widget class="QPushButton" name="changeDirButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>更改目录</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="openDirButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>打开文件夹</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="shortKeyLabel">
             <property name="text">
              <string>快捷按键</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="sendMessageLable">
               <property name="minimumSize">
                <size>
                 <width>112</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>发送消息</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="topMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="hotKeyScreenShot">
               <property name="minimumSize">
                <size>
                 <width>112</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>截图　　</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="screentShotShortCut">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>设置快捷键</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="topMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="checkHotKey">
               <property name="minimumSize">
                <size>
                 <width>112</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>检测热键</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="hotKeyCheckBox">
               <property name="text">
                <string>热键与其他软件冲突时提醒</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>261</width>
         <height>144</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>localStorageFileLineEdit</tabstop>
  <tabstop>changeDirButton</tabstop>
  <tabstop>openDirButton</tabstop>
  <tabstop>hotKeyCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>TNLocaleStorageSettingWidget</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
